### PR TITLE
Fix team numbers on player disconnect

### PR DESCRIPTION
**Pull request by cerebral-314**

### CODE REVIEW

**Comment by SavageGameDesign** [comment]
Task linked: [CU-2p11avq Users do not clear from team array when they leave a server](https://app.clickup.com/t/2p11avq)

File: mission/para_player_loaded_client.sqf [hunk 1]
```diff
@@ -50,15 +50,6 @@ player addEventHandler ["Respawn", _fnc_disableChatter];
 //This way they can still exit if they want.
 player enableSimulation false;
 
-//Assign player to Mike Force if they aren't in a group or failed to join one.
```
Comment (Alablm) [inline mission/para_player_loaded_client.sqf:53]: Why did we get rid of this fallback?

File: mission/para_player_loaded_client.sqf [hunk 2]
```diff
@@ -50,15 +50,6 @@ player addEventHandler ["Respawn", _fnc_disableChatter];
...
-private _playerGroup = player getVariable ["vn_mf_db_player_group", "FAILED"];
-if(_playerGroup == "FAILED") then // Should be the only time this check is needed.
-{
-	[player, "MikeForce"] call vn_mf_fnc_force_team_change;
-} else {
-	[player, _playerGroup] call vn_mf_fnc_force_team_change;
-};	
-
```
Comment (Alablm) [inline mission/para_player_loaded_client.sqf:61]: Why did we get rid of this fallback?

Comment (cerebral-314) [reply mission/para_player_loaded_client.sqf:61]: Upon joining the game... you would join two teams if you weren't in Mike Force originally.
The client shouldn't ever force a team change anyway. Now that I think about it... I'm not sure why I added this to begin with.

Was causing the weird team numbers.

**Review by Alablm — APPROVED** [review]

**Pull request merged**